keras_cv/models/mobilenet_v3.py

```diff
@@ -0,0 +1,534 @@
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        minimalistic: in addition to large and small models this module also
+            contains so-called minimalistic models, these models have the same
+            per-layer dimensions characteristic as MobilenetV3 however, they don't
+            utilize any of the advanced blocks (squeeze-and-excite units, hard-swish,
+            and 5x5 convolutions). While these models are less efficient on CPU, they
+            are much more performant on GPU/DSP.
+        include_top: whether to include the fully-connected layer at the top of the
+            network.  If provided, `num_classes` must be provided.
+        weights: one of `None` (random initialization), or a pretrained weight file
+            path.
+        num_classes: optional number of num_classes to classify images into, only to be
+            specified if `include_top` is True, and if no `weights` argument is
+            specified.
+        pooling: optional pooling mode for feature extraction
+            when `include_top` is `False`.
+            - `None` means that the output of the model will be the 4D tensor output
+                of the last convolutional block.
+            - `avg` means that global average pooling will be applied to the output
+                of the last convolutional block, and thus the output of the model will
+                be a 2D tensor.
+            - `max` means that global max pooling will be applied.
+        dropout_rate: a float between 0 and 1 denoting the fraction of input units to
+            drop, defaults to 0.2.
+        classifier_activation: the activation function to use, defaults to softmax.
+        include_rescaling: whether or not to Rescale the inputs.If set to True,
+            inputs will be passed through a `Rescaling(scale=1.0 / 127.5, offset=-1.0)`
+            layer, defaults to True.
+        name: (Optional) name to pass to the model. Defaults to "{name}".
+
+    Returns:
+        A `keras.Model` instance.
+"""
+
+
+def Depth(divisor=8, min_value=None, name=None):
+    """Ensure that all layers have a channel number that is divisble by the `divisor`.
+
+    Args:
+        divisor: integer, the value by which a channel number should be divisble,
+            defaults to 8.
+        min_value: float, minimum value for the new tensor.
+        name: string, layer label.
+
+    Returns:
+        a function that takes an input Tensor representing a Depth layer.
+    """
+    if name is None:
+        name = f"depth_{backend.get_uid('depth')}"
+
+    if min_value is None:
+        min_value = divisor
+
+    def apply(x):
+        new_x = max(min_value, int(x + divisor / 2) // divisor * divisor)
+
+        # make sure that round down does not go down by more than 10%.
+        if new_x < 0.9 * x:
```

Comment:
isn't depth just a function?  I don't see any use of name, or any Keras operations.  I think depth is just a utility function, and shouldn't be written as a closure.

Reply:
Yes, it's just a utility function